src/sentry/services/hybrid_cloud/organization_mapping/__init__.py

```diff
@@ -34,15 +27,26 @@ class RpcOrganizationMapping:
     customer_id: Optional[str] = None
 
 
-@dataclass
-class RpcOrganizationMappingUpdate(PatchableMixin["Organization"]):
-    organization_id: int = -1
-    name: Unset[str] = UnsetVal
-    customer_id: Unset[str] = UnsetVal
+class RpcOrganizationMappingUpdate(TypedDict):
+    """A set of values to be updated on an OrganizationMapping.
+
+    An absent key indicates that the attribute should not be updated. (Compare to a
+    `"customer_id": None` entry, which indicates that `customer_id` should be
+    overwritten with a null value.)
+    """
 
-    @classmethod
-    def from_instance(cls, inst: Organization) -> "RpcOrganizationMappingUpdate":
-        return cls(**cls.params_from_instance(inst), organization_id=inst.id)
+    name: str
+    customer_id: Optional[str]
+
+
+def update_organization_mapping_from_instance(
+    organization: Organization,
+) -> RpcOrganizationMappingUpdate:
+    attributes = {
+        attr_name: getattr(organization, attr_name)
+        for attr_name in RpcOrganizationMappingUpdate.__annotations__.keys()
+    }
+    return RpcOrganizationMappingUpdate(**attributes)  # type: ignore
```

Comment:
This could probably be generalized for all `TypedDict`s? Didn't want to overengineer it, though.

Alternatively, we could just do
```python
return {"name": organization.name, "customer_id": organization.customer_id}
```
as long as we're not too worried about someone adding a field to `RpcOrganizationMappingUpdate` without noticing they need to update this function as well.

Reply:
Yeah, I think there is a future where this needs generalization, a bit.  It's tricky, because *some* model updates will be a simple mapping of attributes nearly as is, but some won't (when we get to replicating denormalized models, for instance).  We can save on it for now since this isn't even fully hooked up.